common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/responses/signin/SignInChallengeApiResponse.kt

```diff
@@ -52,6 +52,18 @@ class SignInChallengeApiResponse(
     @SerializedName("error_uri") val errorUri: String?,
 ): IApiResponse(statusCode, correlationId) {
 
+    override fun toUnsanitizedString(): String {
+        return "SignInChallengeApiResponse(statusCode=$statusCode, " +
+                "correlationId=$correlationId, challengeType=$challengeType, " +
+                "bindingMethod=$bindingMethod, challengeTargetLabel=$challengeTargetLabel, " +
+                "challengeChannel=$challengeChannel, codeLength=$codeLength, interval=$interval, " +
+                "error=$error, errorDescription=$errorDescription, errorCodes=$errorCodes, " +
+                "errorUri=$errorUri)"
+    }
+
+    override fun toString(): String = "SignInChallengeApiResponse(statusCode=$statusCode, " +
+            "correlationId=$correlationId"
```

Comment:
nit: Missing bracket and some parameters. ("challengeChannel=$challengeChannel, codeLength=$codeLength, interval=$interval")